docs/changelog-fragments.d/243.feature.md

```diff
@@ -0,0 +1,3 @@
+Implemented falling back to `--syntax-check` from ansible-lint covering all
+scenarios and keeping in mind different patterns of responses sent by
```

Comment:
This "covering all scenarios and keeping in mind different patterns" sounds like a marketing speech. It's best to make this message accurate and clarify what this actually means for the end-users. Maybe with examples.

Reply:
Okay.
Updated.